google/cloud/status.h

```diff
@@ -102,7 +105,9 @@ class Status {
 };
 
 inline std::ostream& operator<<(std::ostream& os, Status const& rhs) {
-  return os << rhs.error_message() << " [" << rhs.status_code()
+  return os << rhs.error_message()
+            << StatusCodeToString(static_cast<StatusCode>(rhs.status_code()))
+            << " [" << rhs.status_code()
```

Comment:
If this is useful, perhaps StatusCode operator<<() should be doing it.

Reply:
That gave me an idea, the only case where this is useful is when the value is not one of the well known values. I changed the `StatusCodeToString` function to return `UNEXPECTED_STATUS_CODE=%d` in this case, what do you think?

Reply:
SGTM.  I actually had that as part of my draft suggestion but removed it when I saw the (what would have been redundant) rendering of the code here.  I think I prefer falling on the other side of the fence, as you've done here.